src/compound/lens/LensStorage.sol

```diff
@@ -28,6 +29,7 @@ abstract contract LensStorage is ILens, Initializable {
     IMorpho public immutable morpho;
     IComptroller public immutable comptroller;
     IRewardsManager public immutable rewardsManager;
+    ILensExtension internal immutable lensExtension;
```

Comment:
So we will need to upgrade the full lens for any change in the extension. It makes sense for simplifying the code, I just want to make sure that everyone is aware.